Polygon_mesh_processing/test/Polygon_mesh_processing/test_pmp_locate.cpp

```diff
@@ -537,33 +583,44 @@ struct Locate_with_AABB_tree_Tester // 2D case
     // sanitize otherwise some test platforms fail
     PMP::internal::snap_location_to_border(loc, g, FT(1e-7));
```

Comment:
From below, there is snapping, which should take care of the approximations that come with an inexact construction kernel, and so I would not expect the EPECK filtering to be required. Did you in practice observe issues with those calls?

Reply:
@soesau Did this also give exceptions when reverted?

Reply:
All PMP::is_in_face in Locate_with_AABB_tree_Tester::test fail.
The first asserts after snap_location_to_border do not fail with Epick.

Reply:
@MaelRL resolved?